src/services/requetsToEndpoint.ts

```diff
@@ -0,0 +1,38 @@
+import { getToken } from "./generateToken";
+import Track from "../models/track.inteface";
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (typo):** There seems to be a typo in the file name `track.inteface.ts`. Typically, interface files are named with the suffix 'interface', suggesting this might be a typo. Please verify and correct the file name to `track.interface.ts` for consistency and clarity.